packages/govern-console/src/containers/HomePage/HomePage.tsx

```diff
@@ -47,12 +48,15 @@ const HomePage = ({ ...props }) => {
               updateSelectedDao={updateSelectedDaoAndPushToHistory}
             />
           </Route>
-          <Route exact path="/:daoName">
+          <Route exact path="daos/:daoName">
```

Comment:
Please change the route.push method as well otherwise clicking on a DAO card will break the app.